client/galaxy/scripts/mvc/workflow/workflow.js

```diff
@@ -0,0 +1,201 @@
+/** Workflow view */
+define( [ 'mvc/form/form-view',
+          'mvc/ui/ui-misc',
+          'mvc/ui/ui-select',
+          'mvc/tool/tool-form-composite' ], function( Form, Ui, Select, ToolForm ) {
```

Comment:
Actually I rebuilt the client after removing the imports from the header of workflow.js to verify they are not needed and now I have all those static files modified. Shall I commit all these static files along with conflicting files?

### This branch has conflicts that must be resolved
```
Conflicting files
static/scripts/bundled/libs.bundled.js
static/scripts/bundled/libs.bundled.js.map
```